### ISSUE TITLE
Results obtained from various turbines?

### ISSUE THREAD

**Issue by koshatnik** [turbinesFoam/turbinesFoam#248]
I have modeled two different wind turbines; MEXICO turbine and DTU 10MW Reference turbine. These are wind turbines which has detailed blade and airfoil data available.
However, I am getting unrealistic C_p results, like 1.12 for MEXICO and 0.72 for DTU 10MW (these are converged results). I would like to know if anyone else has run with their settings and share their results.

**Comment by tLashbrooke** [comment]
AFK at the moment but will be able to answer tomorrow , in the mean time
could you tell me the radius of the turbines, the number of cells on the
turbine cellset and the edge length?

On 8 Mar 2018 19:40, "Thomas Lashbrooke" <thomas.lashbrooke@myport.ac.uk>
wrote:



Thomas Lashbrooke

**Comment by koshatnik** [comment]
Sure, first of all, these two turbines are CW rotating, but since CW rotation ( axis(1 0 0) ) does not work with endEffects on (this is a bug), I am running the simulations CCW.

MEXICO: 
Rotor radius: 2.25m 
Created cellSet turbine
    Applying source cylinderToCell
    Adding cells with centre within cylinder, with p1 = (-0.5 0 0), p2 = (0.5 0 0) and radius = 2.25
    cellSet turbine now size 1968
Created cellZoneSet turbine
    Applying source setToCellZone
    Adding all cells from cellSet turbine ...
    cellZoneSet turbine now size 1968
This has 8 cell thickness.

DTU10MW:
Rotor radius: 89.15m
Created cellSet turbine
    Applying source cylinderToCell
    Adding cells with centre within cylinder, with p1 = (-5 0 0), p2 = (5 0 0) and radius = 90
    cellSet turbine now size 8120
Created cellZoneSet turbine
    Applying source setToCellZone
    Adding all cells from cellSet turbine ...
    cellZoneSet turbine now size 8120
This has 2 cell thickness.

I did not model hub and tower.

**Comment by koshatnik** [comment]
I can also share my case directory and datasheets for the turbines. Another pair of eyes would be good for noticing the mistake.

**Comment by petebachant** [comment]
Note that you do want your `cellZones` to be a bit larger than the rotor to account for the Gaussian smearing of force. I doubt this is causing the issue though. Are you running incompressible?

**Comment by koshatnik** [comment]
Yes, I am running `pimpleFoam`. I am aware of the issue you are pointing, but I got very similar results for larger `cellZones`. (I have been struggling with these cases for a week now and tried a lot of different combinations). I doubted the correctness of my `elementProfiles` settings, and even tried the same airfoil profile used in the tutorial. That also did not work. I also wait for cp, ct values to reach a terminal value.

**Comment by tLashbrooke** [comment]
I'm certain that your mesh isn't fine enough, I'd look at Petes NTHU HAWT
case to get a rough idea of what you need. Also the paper cited in the
readme shows that decreasing spatial grid distance is the best way to
establish grid independence, then also consider varying the time step using
iterations per revolution. My case needed a good 300 per rev with a TSR of
5.0955


On 8 Mar 2018 20:35, "koshatnik" <notifications@github.com> wrote:

Oops, saw your advise after posting. My apologies, I am not very good at
using GitHub yet. Give me some time to learn how to do that please.
And yes, I specifically paid attention to matching the velocities.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/turbinesFoam/turbinesFoam/issues/248#issuecomment-371598587>,
or mute the thread

**Comment by koshatnik** [comment]
After reading your post, I have run NTNU-HAWT-turbinesFoam case of Mr. Bachant's. First run was with the provided settings, and then I have disabled the snappyHexMesh refinement (used the coarsest available blockMesh grid) and increased the time step to max allowable for Courant = 0.95. Resultant C_p values were really close. I have used very fine grids and small time steps for my case but the result did not change before. Mr. Bachant's case did not give divergent results for coarse settings. I will again try refining, but at the same time I suspect that some other settings might be wrong (even though I got them from papers).
I also noticed that there is a https://github.com/petebachant/MEXICO-turbinesFoam model which has been inactive for 2 years. Can this provide any improvement?

**Comment by petebachant** [comment]
That repo is really just a placeholder. I wanted to make a case for the MEXICO rotor but just never had the time.

Keep in mind I usually run with a fixed timestep so `maxCo` doesn't have an effect on `deltaT`.

**Comment by kkpal3** [comment]
> Actuall, just let me upload the case files.
> [axialFlowMexico-v2.tar.gz](https://github.com/turbinesFoam/turbinesFoam/files/1794521/axialFlowMexico-v2.tar.gz)
> [axialFlowDTU_v1.tar.gz](https://github.com/turbinesFoam/turbinesFoam/files/1794522/axialFlowDTU_v1.tar.gz)


Hi hcOnel, I'm new to wind turbine modeling. May I ask where did obtain the DTU turbine blade information in the case files you uploaded in this issue?
